modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -2448,6 +2448,24 @@ private ObjectDefinition _publishObjectDefinition(
 
 		objectDefinition.setStatus(WorkflowConstants.STATUS_APPROVED);
 
+		for (ObjectField of : objectFields) {
```

Comment:
@Aquiiiles rename it to `for (ObjectField objectField : objectFields) {`

[Variable name is the type name](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#%5BinlineExtension%5DVariable-name-is-the-type-name).